setup.cfg

```diff
@@ -36,7 +36,7 @@ omit =
     versioneer.py
     unyt/_version.py
     unyt/_on_demand_imports.py
-    unyt/_testing.py
```

Comment:
If we're exposing this publicly I don't think we can get away with excluding it from test coverage anymore.

Reply:
Added `tests/test_unyt_testing.py`.